client/src/components/DatasetProjectAdditionalOptions.js

```diff
@@ -1,36 +1,90 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { Box, CheckBox, Text } from 'grommet'
+import { useDatasetManager } from 'hooks/useDatasetManager'
 import { config } from 'config'
 import { HelpLink } from 'components/HelpLink'
 import { InfoText } from 'components/InfoText'
 import { Link } from 'components/Link'
 import { FormField } from 'components/FormField'
 import { WarningAnnDataMultiplexed } from 'components/WarningAnnDataMultiplexed'
 
-export const DatasetProjectAdditionalOptions = ({ project }) => {
+export const DatasetProjectAdditionalOptions = ({
+  project,
+  samples,
+  selectedModalities,
+  additionalOptions,
+  setAdditionalOptions
+}) => {
+  const {
+    myDataset,
+    userFormat,
+    isProjectBulkIncluded,
+    isProjectExcludedMultiplexed,
+    isProjectSingleCellMerged
+  } = useDatasetManager()
+
   const {
     has_bulk_rna_seq: hasBulkRnaSeq,
     has_multiplexed_data: hasMultiplexed,
+    has_single_cell_data: hasSingleCellData,
     includes_merged_sce: includesMergedSce,
     includes_merged_anndata: includesMergedAnnData
   } = project
 
-  const isMergedObjectsAvailable = includesMergedSce || includesMergedAnnData
-  const isExcludeMultiplexedAvailable = false
+  const isMergedObjectsAvailable =
+    (includesMergedSce || includesMergedAnnData) &&
+    (selectedModalities.includes('SINGLE_CELL') ||
+      (selectedModalities.length === 0 && hasSingleCellData && !hasMultiplexed))
```

Comment:
I dont think we need to do this much logic here.

1. Depending on the format of the dataset we can check the appropriate `includes_merged_sce` or `includes_merged_ann_data`.
2. If the user has no format you can check either of them.
3. `hasMultiplexed` and `hasSingleCellData` do not need to be checked here. that information is contained in `includes_merged_sce` etc.
4. You only need to check `hasMultiplexed` to show the explanation when merged is not available.